code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -395,8 +404,8 @@
 	if(contents.len) //If they used the soulstone on someone else in the meantime
 		return 0
 	T.ckey = chosen_ghost.ckey
-	for(var/obj/item/W in T)
-		T.unEquip(W)
+	if(ishuman(T))

```

Comment:
why is this check here?

Reply:
If I just do the T.unEquip(W) loop on something without making sure it's human, like on say a cyborg, it will dump out all of it's internal pieces it shouldn't. 

At least that's what I assume was causing this issue.

![sacrifice0](https://user-images.githubusercontent.com/47765135/66591479-06cd8c80-eb61-11e9-92ed-2ac62533ee46.png)